cpp-ch/local-engine/Storages/SubstraitSource/FormatFile.cpp

```diff
@@ -55,25 +55,20 @@ FormatFile::FormatFile(
     : context(context_), file_info(file_info_), read_buffer_builder(read_buffer_builder_)
 {
     PartitionValues part_vals = GlutenStringUtils::parsePartitionTablePath(file_info.uri_file());
-    String partition_values_str = "[";
     for (size_t i = 0; i < part_vals.size(); ++i)
     {
         const auto & part = part_vals[i];
         partition_keys.push_back(part.first);
         partition_values[part.first] = part.second;
-        if (i > 0)
-            partition_values_str += ", ";
-        partition_values_str += part.first + "=" + part.second;
     }
-    partition_values_str += "]";
+
     LOG_INFO(
         &Poco::Logger::get("FormatFile"),
-        "Reading File path: {}, format: {}, range: {}, partition_index: {}, partition_values: {}",
+        "Reading File path: {}, format: {}, range: {}, partition_index: {}",
```

Comment:
Why removes the print of partition_values? Vanilla Spark also has the behaviour.

Reply:
I thought the file path contains all the information we want to know about partitions..